src/rules/cyclomaticComplexityRule.ts

```diff
@@ -105,6 +105,7 @@ function walk(ctx: Lint.WalkContext<{ threshold: number }>): void {
 function increasesComplexity(node: ts.Node): boolean {
     switch (node.kind) {
         case ts.SyntaxKind.CaseClause:
+            return (node as ts.CaseClause).statements.length > 0;
```

Comment:
btw. if revert your change here to make them equal again, you probably need to disable the rule in almost all `ts.forEachChild` callbacks in this repo